test/main.fmf

```diff
@@ -0,0 +1,30 @@
+# Only common dependencies that are NOT required to run podman-tests.sh are
+# specified here. Everything else is in podman-tests.sh.
+require:
+    - make
+    - policycoreutils
+
+/basic_check:
+    summary: Run basic checks
+    test: make basic_check
+    tag: always
+
+/podman_e2e_test_upstream:
+    summary: Run SELinux specific Podman e2e tests on upstream PRs
+    test: make podman_e2e_test_upstream
+    tag: upstream
```

Comment:
```suggestion
/upstream:
    tag: upstream
/upstream/podman_e2e_test:
    summary: Run SELinux specific Podman e2e tests on upstream PRs
    test: make podman_e2e_test_upstream
```

Other things you can do is to use the [`inheritance`](https://fmf.readthedocs.io/en/stable/features.html#inheritance) and [`elasticity`](https://fmf.readthedocs.io/en/stable/features.html#elasticity). Up to you how you think it will be easier to organize.